audformat/core/utils.py

```diff
@@ -285,6 +285,32 @@ def duration(
     return (ends - starts).sum()
 
 
+def hash(
+        obj: typing.Union[pd.Index, pd.Series, pd.DataFrame],
+) -> str:
+    r"""Create hash from object.
+
+    Objects with same entries in a different order,
+    will produce the same hash string.
```

Comment:
Should we add a test for this or simply trust `pandas` here?

Reply:
This is not really related to `pandas` but the fact that we calculate the sum of the hashes we get for each row.

But since it's never a bad idea to have tests, so I added some now :)

I also rephrased to: 

![image](https://user-images.githubusercontent.com/10383417/142039796-15982637-442b-480e-80ac-43937b396759.png)

Reply:
Great